packages/integrations/stores/gcp-secret-manager-config-store/src/gcp-secret-manager.ts

```diff
@@ -0,0 +1,65 @@
+import { SecretManagerServiceClient } from '@google-cloud/secret-manager';
```

Comment:
you should add this package as a dependency.

Reply:
@davidmeirlevy 
Found the version in the package.json from the old store json file.
seems like according to the checks it's not the same as the one specified in the validation.
where can I see the correct version for this dependency?

Reply:
"@google-cloud/secret-manager": "^5.4.0",

Reply:
so the other import was the issue maybe